### ISSUE TITLE
protobuf message naming document

### ISSUE THREAD

**Issue by mapleray** [icewind1991/steam-vent#29]
Hi, I'm new to this, and I'm hoping to do some work interacting with steam/dota2 by using this library, and I'm a bit confused about the protobuf file right now, is there any documentation about `kind`  in `protobuf/build`  for parsing protobuf file? 
 I can guess the usage from the code in the `protobuf/build`, but I can't understand it well. 
Is there a general introduction to the naming of message/enum variables in protobuf files?
Thx

**Comment by icewind1991** [comment]
You shouldn't need to deal with the protobufs yourself, the library comes with the dota2 protobufs/rust structs (behind a feature flag)

**Comment by mapleray** [comment]
> You shouldn't need to deal with the protobufs yourself, the library comes with the dota2 protobufs/rust structs (behind a feature flag)

yeah... but just want to know why parsing like this in `protobuf/build`  😄😄
